docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -163,25 +167,29 @@ the configuration are described in the following listings:
 	}
 
 	asciidoctor { <6>
-		inputs.dir snippetsDir <7>
-		dependsOn test <8>
+		dependsOn test <7>
+		configurations 'asciidoctorExt'
```

Comment:
I think it'd be worth adding a callout that explains what's happening here.

Reply:
Done!